packages/lesswrong/components/common/SearchBar.tsx

```diff
@@ -180,7 +180,7 @@ const SearchBar = ({onSetIsActive, searchResultsArea, classes}: {
         )}>
           {alignmentForum && <VirtualMenu attribute="af" defaultRefinement="true" />}
           <div onClick={handleSearchTap}>
-            <SearchIcon className={classes.searchIcon}/>
+            <ForumIcon icon="Search" className={classes.searchIcon}/>
```

Comment:
I don't suppose we could get this to have a darken-on-hover effect like the other menu items?